sdk/eventgrid/azure-messaging-eventgrid-namespaces/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
+- Azure EventGrid client library for Java. This package contains Microsoft Azure EventGrid client library.
```

Comment:
Should we make this more specific about namespaces given that we have another library which matches this description?